docs/static/spec/rasa.yml

```
@@ -880,16 +880,19 @@ components:
     200Story:
       description: Success
       content:
-        text/markdown:
+        yml:
           example: >-
-            ## story_00055028
-
-            * greet: hello
-             - utter_ask_howcanhelp
-            * inform: I'm looking for a [moderately priced](price:moderate)
-            [Indian](cuisine) restaurant for [two](people) people
-             - utter_on_it
-             - utter_ask_location
+            - story: 00055028
+              steps:
+              - user: |
+                  hello
+                intent: greet
+              - action: utter_ask_howcanhelp
+              - user: |
+                  I'm looking for a [moderately priced](price:moderate) [Indian](cuisine) restaurant for [two](people) people
```

This format for entities is deprecated (or even gone I think), see [the doc here](https://rasa.com/docs/rasa/training-data-format#entities). I think you could go with:
```suggestion
                  I'm looking for a [moderately priced]{"entity": "price", "value": "moderate"} [Indian]{"entity": "cuisine"} restaurant for [two]({"entity": "people"}) people
```

---

I see, thanks for the pointer! 😃